modules/jfx.incubator.input/src/main/java/jfx/incubator/scene/control/input/InputMap.java

```diff
@@ -276,11 +280,15 @@ public void registerKey(KeyBinding k, FunctionTag tag) {
     }
 
     /**
-     * Unbinds the specified key binding.
+     * Unregister the specified key binding.
+     * Calling this method will disable any mappings made with
+     * {@link #register(KeyBinding, Runnable)},
+     * {@link #registerKey(KeyBinding, FunctionTag)},
+     * or registered by the skin.
      *
      * @param k the key binding
      */
-    public void unbind(KeyBinding k) {
+    public void unregister(KeyBinding k) {
         map.put(k, NULL);
```

Comment:
Why does this add a `NULL` rather than remove the mapping? I'm trying to understand the difference between this method and `resetKeyBindings` (maybe this is related to my earlier question above?). One thing to consider is that allowing `NULL` values means that there is no way to tell the difference between a key with a `NULL` value and a key that isn't in the map without calling `map.contains`, which you don't expose.

Reply:
I see. In that case, unregister is not a good name, since it is _not_ the inverse of register. What it is really doing is registering a no-op binding. In which case... why is it needed at all? The application can just as easily do this:

```
    register(keyBinding, () -> {});
```

If there really is a compelling reason to keep this method, then we need to think of a better name. Maybe "disable" or "disableKeyBinding"?